### PR TITLE
PR - Review assignment 

### PR DESCRIPTION
**Pull request by martinaxgloria**

### CODE REVIEW

File: src/navigation-control.thrift [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    * @return true/false on success/failure.
+    */
+   bool track_skeleton(1:string skeleton_tag);
+
+   /**
+    * Query if navigation is underway.
+    * @return true if navigation is being currently performed.
+    */
+   bool is_navigating();
+}
```
Comment (pattacini) [inline src/navigation-control.thrift:117]: This part is appended instead of being integrated with the first contribution.

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @martinaxgloria 

The Git history looks pretty neat 👍🏻 

However, the navigation features ended up being merged incorrectly. There are repetitions in the resulting module. Give a glance at the inline comments for that.

**Comment by martinaxgloria** [comment]
Thank you @pattacini, now it's integrated and not appended.

**Comment by pattacini** [comment]
> Thank you @pattacini, now it's integrated and not appended.

Pitch perfect 👍🏻 
You can now merge the PR.

Consider the 3 different strategies proposed by GitHub to merge the PR, which can be selected with the drop-down menu offered by the <kbd>green button</kbd>.

**Pull request merged**